polynote-kernel/src/main/scala/polynote/messages/Messages.scala

```diff
@@ -234,10 +241,21 @@ final case class HandleData(
   handleType: HandleType,
   handle: Int,
   count: Int,
-  data: List[ByteVector32]
+  data: Array[ByteVector32]
 ) extends Message
 
 object HandleData extends MessageCompanion[HandleData](17)
 
-final case class CancelTasks(path: ShortString) extends Message
-object CancelTasks extends MessageCompanion[CancelTasks](18)
\ No newline at end of file
+/********************************************************
+ ** Specifically for streams of structs (i.e. tables)  **
+ *******************************************************/
+
+final case class ModifyStream(path: ShortString, fromHandle: Int, ops: TinyList[TableOp], newRepr: Option[StreamingDataRepr]) extends Message
+object ModifyStream extends MessageCompanion[ModifyStream](19) {
+  import TableOpCodec.tableOpCodec
+  import ValueReprCodec.streamingDataReprCodec
```

Comment:
IntelliJ marks these inputs as unused but it looks like they are necessary - maybe we should add a comment when it's being stupid so we don't try to clean it up later?